pkg/cli/common.go

```diff
@@ -147,9 +147,9 @@ var (
 			Name:  "quiet, q",
 			Usage: "refrain from announcing build instructions and image read/write progress",
 		},
-		cli.BoolFlag{
+		cli.BoolTFlag{
 			Name:  "rm",
-			Usage: "Remove intermediate containers after a successful build. The build process does not currently support caching so this is a NOOP.",
+			Usage: "Remove intermediate containers after a successful build (default true)",
```

Comment:
I was suprised to see this defaulted to true.  Does this mean that if I run 
buildah bud --layers .
If the build is successful and I go add a new line to the Dockerfile at the end, and run the same command again, it will go through every step, nothing will be cached?

Reply:
No, this just means that the containers created during the build process will be removed. After each stem the container is committed to an intermediate image so the cache uses those images. The intermediate containers don't really come into play unless you want to keep them around for debugging I guess.